examples/select-features.js

```diff
@@ -61,6 +61,11 @@ var changeInteraction = function() {
   }
   if (select !== null) {
     map.addInteraction(select);
+    select.on('select', function(e) {
+      $('#status').html('&nbsp;' + e.features.getLength() +
```

Comment:
Can't you use `e.target.getFeatures().getLength()` or `select.getFeatures().getLength()` here?